com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -10,8 +10,11 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Added
 
+- Added a protected virtual method `NetworkTransform.OnInitialize(ref NetworkTransformState replicatedState)` that just returns the replicated state reference.
```

Comment:
This seems unrelated to the PR, is it supposed to be here?

Reply:
Well, it makes it easier if we merge this into the changelog now since that adjustment was made to #2657

Reply:
I could take it out... it should just collapse when merged.

Reply:
Don't think we need it in two PRs... but either way, can you add the PR number for this changelog entry?

Reply:
Eh.. you are right... no point in having it in two places and it might be confusing being in this PR... your initial gut reaction is probably the right way...
"This is the way..."